packages/css/src/components/mega-menu/mega-menu.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+
+.amsterdam-mega-menu {
+  padding-block: 3.5rem 4.5rem;
+  padding-inline: 1.5rem;
+
+  @media screen and (min-width: $amsterdam-breakpoint-wide) {
+    padding-block: 5rem 7.5rem;
+    padding-inline: 5.24rem;
+  }
+}
+
+.amsterdam-mega-menu .amsterdam-grid__cell {
+  .amsterdam-heading:nth-child(n + 1) {
+    padding-block-start: 3.5rem;
+  }
+}
+
+.amsterdam-mega-menu .amsterdam-unordered-list {
+  --amsterdam-grid-gap: var(--amsterdam-grid-spacious-gap);
+
+  column-gap: var(--amsterdam-grid-gap);
+  columns: 20rem auto;
```

Comment:
This feels a bit magic number-y. Couldn't we create a component with a configurable column count per breakpoint?

Reply:
Let's discuss which approach we want to take here tomorrow